src/System.Runtime.Analyzers/UnitTests/DoNotRaiseReservedExceptionTypesTests.cs

```diff
@@ -10,6 +10,119 @@ namespace System.Runtime.Analyzers.UnitTests
 {
     public class DoNotRaiseReservedExceptionTypesTests : DiagnosticAnalyzerTestBase
     {
+        [Fact]
+        public void CreateSystemNotImplementedException()
+        {
+            VerifyCSharp(@"
```

Comment:
Putting equivalent C# and VB tests in a single unit tests seems like a good approach to me.